app/views/browse/deployment.html

```diff
@@ -1,141 +1,266 @@
+                                      scale-fn="scale(replicas)"
+                                      deployment="deployment"></replicas>
+                            <span ng-if="autoscalers.length">(autoscaled)</span>
+                            <div ng-if="deployment.status.updatedReplicas">
+                              {{deployment.status.updatedReplicas}} up to date
+                            </div>
+                            <div ng-if="deployment.status.availableReplicas || deployment.status.unavailableReplicas">
+                              <span ng-if="deployment.status.availableReplicas">{{deployment.status.availableReplicas}} available<span ng-if="deployment.status.unavailableReplicas">,</span></span>
+                              <span ng-if="deployment.status.unavailableReplicas">{{deployment.status.unavailableReplicas}} unavailable</span>
+                            </div>
+                          </dd>
+                          <dt>Paused:</dt>
+                          <dd>{{!!deployment.spec.paused}}</dd>
+                        </dl>
+                      </div>
+                      <div class="col-lg-6">
+                        <h3>Configuration</h3>
+                        <dl class="dl-horizontal left">
+                          <dt>Selectors:</dt>
+                          <dd>
+                            <selector selector="deployment.spec.selector"></selector>
+                          </dd>
+                          <dt>Strategy:</dt>
+                          <dd>{{deployment.spec.strategy.type | sentenceCase}}</dd>
+                          <dt ng-if-start="deployment.spec.strategy.rollingUpdate">Max Unavailable:</dt>
+                          <dd>
+                            {{deployment.spec.strategy.rollingUpdate.maxUnavailable}}
+                            <span data-toggle="tooltip"
+                              title="The maximum number of pods that can be unavailable during the update process."
+                              class="pficon pficon-help text-muted small"></span>
+                          </dd>
+                          <dt>Max Surge:</dt>
+                          <dd ng-if-end>
+                            {{deployment.spec.strategy.rollingUpdate.maxSurge}}
+                            <span data-toggle="tooltip"
+                              title="The maximum number of pods that can be created above the desired number of pods."
+                              class="pficon pficon-help text-muted small"></span>
+                          </dd>
+                          <dt>Min Ready:</dt>
+                          <dd>
+                            {{deployment.spec.minReadySeconds || 0}} sec
+                            <span data-toggle="tooltip"
+                              title="The minimum number of seconds for which a newly created pod should be ready without any of its containers crashing for it to be considered available."
```

Comment:
hmm this sentence is hard to parse, isnt the definition of ready that the containers are started, so if its ready for X seconds then it was ready for X seconds with no containers crashing?
Should this just be:
`The minimum number of seconds a new pod must be ready before it will be considered available.`